deployment/debian-package-x64/pkg-src/rules

```diff
@@ -2,6 +2,8 @@
 CONFIG := Release
 TERM := xterm
 SHELL := /bin/bash
+WEB_TARGET := $(CURDIR)/MediaBrowser.WebDashboard/jellyfin-web
+WEB_VERSION := $(shell sed -n -e 's/^version: "\(.*\)"/\1/p' $(CURDIR)/build.yaml)
```

Comment:
what is the way to override `WEB_VERSION` now?

Reply:
There is no way to. It either grabs the "current version", or master, or fails.